apps/cli-gui/src/styles.scss

```diff
@@ -1,7 +1,7 @@
 /* You can add global styles to this file, and also import other style files */
-@import './styles/material';
-@import './styles/core/reset';
-@import './styles/core/variables';
+@import 'material';
+@import 'core/reset';
+@import 'core/variables';
```

Comment:
I'm expecting a single import from the ui lib after these changes, and a single `theme` mixin that should be used here (do the reset inside the `theme` mixin)

Reply:
done